report/topology.go

```diff
@@ -75,10 +76,10 @@ func (n Nodes) Merge(other Nodes) Nodes {
 // given node in a given topology, along with the edges emanating from the
 // node and metadata about those edges.
 type Node struct {
-	Metadata  `json:"-"`
-	Counters  `json:"-"`
+	Metadata  `json:"metadata"`
+	Counters  `json:"counters"`
 	Adjacency IDList        `json:"adjacency"`
-	Edges     EdgeMetadatas `json:"-"`
+	Edges     EdgeMetadatas `json:"edges"`
```

Comment:
I'd stick some omitempty's in there as in https://github.com/weaveworks/scope/commit/b96adf03b70b74c6d6530bc468bdfbba789a55e8